julia-mode.el

```diff
@@ -48,19 +48,15 @@
 
 (defcustom julia-indent-offset 4
   "Number of spaces per indentation level."
-  :type 'integer
-  :group 'julia)
+  :type 'integer)
```

Comment:
is there a reason why we're removing this custom-var from the `julia` group? maybe I missed a discussion about it

Reply:
oh, I see... we're adding `:group 'julia` directly in the mode below. nvm

Reply:
To clarify exactly what's going on here, custom variables are automatically added to the most recent `defgroup`.

Reply:
This is a nice feature, but since not everyone may be aware of it, can you please add a comment explaining this after the `defgroup`?

Reply:
Added comment in b15f13c